java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1249,6 +1249,27 @@ public void zrank(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zremrangebyrank(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+
```

Comment:
It is worth checking **ranks** prior to removing them. `zadd` adds scores and we don't know which ranks are assigned.
At least add a comment

Reply:
You do what ranks are assigned because. Ranks are assigned by score!

Reply:
rank != score (see my comment below)